DependencyInjection/Configuration.php

```diff
@@ -161,6 +161,18 @@ public function getConfigTreeBuilder()
                 ->defaultFalse()
                 ->info('Enables integration with enqueue if set true. Allows resolve image caches in background by sending messages to MQ.')
             ->end()
+            ->booleanNode('templating')
+                ->defaultTrue()
+                ->info('Enables integration with symfony/templating component')
+                ->validate()
+                    ->ifTrue()
+                    ->then(function ($v) {
+                        @trigger_error('Using something else than "false" as "liip_imagine.templating" configuration value has been deprecated since LiipImagineBundle 2.2.', E_USER_DEPRECATED);
```

Comment:
this seems a bit convoluted. i would say "The Symfony templating component has been deprecated and will be removed in Symfony 5. Disable templating by setting "liip_imagine.templating: false" to disable templating support and get rid of this warning.

Reply:
I've updated the message.